bodhi/client/__init__.py

```diff
@@ -374,10 +374,15 @@ def edit(user, password, url, **kwargs):
               type=click.Choice(['newpackage', 'security', 'bugfix', 'enhancement']))
 @click.option('--user', help='Updates submitted by a specific user')
 @click.option('--mine', is_flag=True, help='Show only your updates')
+@click.option('--rows', default=None,
+              type=click.IntRange(1, 100, clamp=True),
```

Comment:
Are we sure that we can't get over 100 pages ?

Reply:
I think that is the limit on number of records per page.